<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="authenticatedNameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="secondAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>List Management</string>
   </property>
  </widget>
  <widget class="QPushButton" name="firstAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>My Lists</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thirdAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calendar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fourthAuthenticatedButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>881</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 60px;
</string>
   </property>
   <property name="text">
    <string>ELISTA, A CC15 FINAL PROJECT</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>-10</y>
     <width>21</width>
     <height>821</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
</string>
   </property>
   <property name="text">
    <string>Hello,</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>1221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;</string>
   </property>
   <property name="text">
    <string>To-do lists, personified.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Academic</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Misc...</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="personalTable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>371</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical, QScrollBar:horizontal {
        background: transparent;
        width: 0px;
        height: 0px;
    }

QToolTip {
    font-size: 18px;
    color: white;
    background-color: #333;
    border: 1px solid #aaa;
    padding: 6px;
    border-radius: 4px;
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="academicTable">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>371</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical, QScrollBar:horizontal {
        background: transparent;
        width: 0px;
        height: 0px;
    }</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="miscTable">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>240</y>
     <width>371</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar:vertical, QScrollBar:horizontal {
        background: transparent;
        width: 0px;
        height: 0px;
    }</string>
   </property>
  </widget>
  <widget class="QComboBox" name="personalBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Priority</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deadline</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Sort By:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Sort By:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Sort By:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="academicBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Priority</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deadline</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="miscBox">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Priority</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Deadline</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
